assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -49,6 +49,9 @@ che.workspace.http_proxy=
 che.workspace.https_proxy=
 che.workspace.no_proxy=
 
+# Configures CA bundle config map name for cluster-wide proxy mode
+che.trusted_ca_bundles_configmap=NULL
```

Comment:
I am really missing the clarity in the docs here. Would be nice to describe what is the cm, why / when it should be set, how cluster-wide proxy mode will work etc.

Reply:
Added more explanation from Anatoly

Reply:
Can you please write a little note that's clearly says this is advanced property and it's not expected to be manually set unless you know really well what you're doing.